tce/src/applibs/LLVMBackend/TCETargetMachine.cc

```diff
@@ -269,6 +269,7 @@ TCEPassConfig::addPreRegAlloc() {
 
 bool
 TCEPassConfig::addPreISel() {
+    addPass(createHardwareLoopsPass());
```

Comment:
Should we add it only if the ADF has one of the supported hardware loop operations?